pandas/tests/plotting/test_frame.py

```diff
@@ -380,6 +380,29 @@ def test_subplots_timeseries(self):
                 self._check_ticks_props(ax, xlabelsize=7, xrot=45,
                                         ylabelsize=7)
 
+    def test_subplots_timeseries_y_axis(self):
+        # GH16953
+        data = {"numeric": np.array([1, 2, 5]),
+                "timedelta": [pd.Timedelta(10, unit="s"),
+                              pd.Timedelta(10, unit="m"),
+                              pd.Timedelta(10, unit="h")],
+                "datetime": [pd.to_datetime("2017-08-01 00:00:00"),
```

Comment:
can you add a datetime with a tz as well

Reply:
sure ^^

Reply:
I think some other types, e.g. Period might fail, if you can add them in *another* test with xfail would be fine (Categoricals *might* work as they area converted to ndarrays)